ikenna-portfolio/portfolio/src/main/webapp/style.css

```diff
@@ -4,20 +4,78 @@
   width: 75%;
 }
 
-.image-format {
-  border: 10px solid black;
-  max-width: 80%;
+#quote-container {
+  margin-top: 20px;
 }
 
-.row {
-  display: flex;
+.bg-1 { 
+  background-color: #7a5b31;
+  color: #ffffff;
+}
+  
+.bg-2 { 
+  background-color: #8d8741;
+  color: #ffffff;
+}
+  
+.bg-3 { 
+  background-color: #b89b1a;
+  color: #070101;
 }
 
+.bg-4 { 
+  background-color: #fbeec1;
+  color: #555555;
+}
+
+.bg-5 { 
+  background-color: #bc986a;
+  color: #ffffff;
+}
+  
 .column {
   flex: 1 1 auto;
   padding: 5px;
 }
 
-#quote-container {
-  margin-top: 20px;
+.container-fluid {
+  padding-bottom: 70px;
+  padding-top: 70px;
+}
+
+.desc {
+  font-size: 16px;
+}
+
+.image-format {
+  border: 10px solid black;
+  max-width: 80%;
+}
+
+.margin {
+  margin-bottom: 45px;
+}
+
+.navbar {
+  border: 0;
+  border-radius: 0;
+  font-size: 12px;
+  letter-spacing: 5px;
+  margin-bottom: 0;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.navbar-nav  li a:hover {
+  color: #1abc9c !important; /*Will prolly delete*/
```

Comment:
Did you need the !important here? If so you should leave a comment why.

Reply:
done, left comment on why (forgot to put it in initially)